site2/docs/functions-overview.md

```diff
@@ -5,30 +5,26 @@ sidebar_label: "Overview"
 ---
 
 This section introduces the following content:
-* [What is Pulsar Functions](#what-is-pulsar-functions)
+* [What are Pulsar Functions](#what-is-pulsar-functions)
```

Comment:
```suggestion
* [What are Pulsar Functions](#16181 -pulsar-functions)
```
Do not forget to update all occurrences where reference `[What is Pulsar Functions](#what-is-pulsar-function)` to `[What are Pulsar Functions](#what-is-pulsar-functions)` in the whole Pulsar doc repo.